src/session1_homework/Exercise3.java

```diff
@@ -0,0 +1,20 @@
+package session1_homework;
+
+public class Exercise3 {
+    // 3. Operations: Write a program that performs various operations (addition, subtraction, multiplication, division, and modulus) on two numbers.
+    // The numbers can be hard-coded into the program. Print the result of each operation.
+
+    public static void main (String[] args){
+
+        double num1 = 7;
+        double num2 = 3;
+
+
+        System.out.println("The sum of the numbers is: " + (num1 + num2));
+        System.out.println("The subtraction of the numbers is: " + (num1 - num2));
+        System.out.println("The multiplication of the numbers is: " + (num1 * num2));
+        System.out.println("The division of the numbers is: " + (num1 / num2));
+        System.out.println("The modus of the numbers is: " + (num1 % num2));
+
```

Comment:
Please review all the classes and delete all the empty rows that we don't need. We can use an empty row to define some logic in the code, but we use just one row for this, not as the first line in a method, and not at the end of the body of a method.